lib/src/pubspec.dart

```diff
@@ -43,6 +43,8 @@ class Pubspec {
 
   Map get environment => (_content['environment'] as Map?) ?? {};
 
+  List<pubspek.Screenshot>? get screenshots => _inner.screenshots;
```

Comment:
Then you can just do: `screenshots.isNotEmpty` going forward.
```suggestion
  List<pubspek.Screenshot> get screenshots => _inner.screenshots ?? [];
```

Reply:
done.